cli/src/main/java/com/box/l10n/mojito/cli/command/param/Param.java

```diff
@@ -94,12 +94,12 @@ public class Param {
   public static final String FILE_TYPE_LONG = "--file-type";
   public static final String FILE_TYPE_SHORT = "-ft";
   public static final String FILE_TYPE_DESCRIPTION =
-      "File type (if none provided it will scan for default formats: XLIFF, XCODE_XLIFF, MAC_STRING, MAC_STRINGSDICT, ANDROID_STRINGS, PROPERTIES, PROPERTIES_NOBASENAME, PROPERTIES_JAVA, RESW, RESX, PO, XTB)";
+      "File type(s), can be a list. If none provided it will scan for default formats: XLIFF, XCODE_XLIFF, MAC_STRING, MAC_STRINGSDICT, ANDROID_STRINGS, PROPERTIES, PROPERTIES_NOBASENAME, PROPERTIES_JAVA, RESW, RESX, PO, XTB";
 
   public static final String FILTER_OPTIONS_LONG = "--filter-options";
   public static final String FILTER_OPTIONS_SHORT = "-fo";
   public static final String FILTER_OPTIONS_DESCRIPTION =
-      "Filter options. Format: option1=value1&option2=value2";
+      "Filter options, can be a list. Format: option1=value1";
```

Comment:
What is the syntax for multiple values? Is it space or comma separated? (An example would still be beneficial there)

Reply:
that should work: `-fo "option1=value1" "option2=value2"`